docs/installation.rst

```diff
@@ -1,22 +1,15 @@
 ==================
-Installation guide
+Installation Guide
 ==================
 
-Melexis Warnings plugin is packaged with pip, but once installed it offers
-direct command line invocation (no need to run python infront). That is why
-installation is as simple as:
+Melexis Warnings plugin is available as a package on PyPI. Once installed, it
+offers direct command line invocation (no need to run python infront).
+Installation is as simple as:
 
 .. code-block:: bash
 
-    # Python2
-    pip2 install mlx.warnings
-
     # Python3
     pip3 install mlx.warnings
 
-    # Install latest and greatest
-    pip install https://github.com/melexis/warnings-plugin/archive/master.zip
```

Comment:
Here I would be even thinking of installing directly from the github master (-e or what it was)... I for sure would not remove it.

Reply:
Ah yes, that is a valid use case. Done in e799fc0ed9cc59bf1caedb7a5bfa805d684fa0e0.